config/form-content/functionaris-new/form.ttl

```diff
@@ -0,0 +1,66 @@
+@prefix form: <http://lblod.data.gift/vocabularies/forms/>.
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix displayTypes: <http://lblod.data.gift/display-types/>.
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+@prefix mandaat: <http://data.vlaanderen.be/ns/mandaat#>.
+@prefix lblodlg: <http://data.lblod.info/vocabularies/leidinggevenden/>.
+@prefix org: <http://www.w3.org/ns/org#>.
+
+ext:persoonF
```

Comment:
Yeah there is indeed a lot of duplication right now, it might be interesting to look if it is possible to have forms that show different fields based on same parameters and non-editable fields if that is possible as well. Or maybe if it is possible to work with subforms that can be included in other forms ...

I would suggest however to merge this now and maybe add a spike story for this?